app/src/components/common/TmBalance.vue

```diff
@@ -13,6 +13,13 @@
       <div class="header-container">
         <h1>Your Balances</h1>
         <div class="buttons">
+          <TmBtn
```

Comment:
shouldn't this only display in the balance row (for staking denoms) if the screen is wide enough?

Reply:
it should behave like the Unstake and Send button no? And Claim Rewards should be moved to Your Stake's BalanceRow but that is another PR.

How will Polkadot users bond their tokens otherwise, if their screen is not wide enough?

Reply:
this logic is displaying it here all the time, no?

Reply:
and in the preview link it never appeared on the balance row

Reply:
OK, now we reached perfection:

![image](https://user-images.githubusercontent.com/40721795/92251928-d788e200-eecd-11ea-9a69-91766cdf9236.png)

Reply:
Fixed! M-m-m-merge :woman_dancing:  ! :fire: